composer.json

```diff
@@ -38,6 +38,9 @@
         "symplify/easy-coding-standard": "^6.0",
         "thecodingmachine/phpstan-strict-rules": "^0.11.0"
     },
+    "suggest": {
+        "ext-simplexml": "*"
```

Comment:
The second part of suggest is not a version, but a string to why. Could you say add that it is needed for the `checkstyle` formatter